### PR TITLE
scroll button added

### PR DESCRIPTION
**Pull request by huzaifa9999**
I have added the scroll to top button again , please check for further merge conflicts 
Thankyou

### CODE REVIEW

**Review by sourabhsikarwar — APPROVED** [review]

**Comment by sourabhsikarwar** [comment]
Is there any specific need for yarn.lock files where npm is already in use? @huzaifa9999

**Comment by huzaifa9999** [comment]
> Is there any specific need for yarn.lock files where npm is already in use? @huzaifa9999

no there isn't , I haven't installed yarn.lock , it just automatically comes when I do npm i

File: src/components/ScrollBtn/ScrollToTop.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react";
+
+
```
Comment (anjula-sack) [inline src/components/ScrollBtn/ScrollToTop.jsx:3]:
```suggestion
```

File: src/components/ScrollBtn/ScrollToTop.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    }, []);
+    const goToTop = () => {
+        window.scrollTo({
+            top: 0,
+            behavior: "smooth",
+        });
+    };
+    return (
+        <div className="top-to-btm">
+            {" "}
```
Comment (anjula-sack) [inline src/components/ScrollBtn/ScrollToTop.jsx:25]:
```suggestion
```

File: src/components/ScrollBtn/ScrollToTop.jsx [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            {showTopBtn && (
+              <button  className="icon-position icon-style"
+                    onClick={goToTop}>
+              <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" >
+  <path strokeLinecap="round" strokeLinejoin="round" d="M12 19.5v-15m0 0l-6.75 6.75M12 4.5l6.75 6.75" />
+</svg>
+
+              </button>
+            )}{" "}
```
Comment (anjula-sack) [inline src/components/ScrollBtn/ScrollToTop.jsx:34]:
```suggestion
            )}
```

File: src/components/ScrollBtn/ScrollToTop.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    return (
+        <div className="top-to-btm">
+            {" "}
+            {showTopBtn && (
+              <button  className="icon-position icon-style"
+                    onClick={goToTop}>
+              <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" >
+  <path strokeLinecap="round" strokeLinejoin="round" d="M12 19.5v-15m0 0l-6.75 6.75M12 4.5l6.75 6.75" />
+</svg>
+
```
Comment (anjula-sack) [inline src/components/ScrollBtn/ScrollToTop.jsx:32]:
```suggestion
```

**Review by anjula-sack — CHANGES_REQUESTED** [review]
There are some formatting issues. @sourabhsikarwar We better setup linters and code formatters.

**Pull request closed without merging**